### ISSUE TITLE
Tree Component, aria-selected attribute not updated

### ISSUE THREAD

**Issue by nuurdev** [primefaces/primereact#1266]
**I'm submitting a ...**
```
[x] bug report
[ ] feature request
[ ] support request

**Codesandbox Case (Bug Reports)**
https://codesandbox.io/s/primereact-test-hh849

**Current behavior**
When clicking on a tree node, the aria-selected attribute is not updated (always false)

**Expected behavior**
When clicking on a tree node, the aria-selected attribute should be set to true (I am trying to style the selected node)

* **React version:**
16.12.0

* **PrimeReact version:**
3.4.0

**Comment by nuurdev** [comment]
The Tree component on the [docs](https://www.primefaces.org/primereact/showcase/#/tree) has the same issue

**Comment by mertsincan** [comment]
Hi @nuurdev,

It works fine for me. Please see selectable tree demo;
https://www.primefaces.org/primereact/showcase/#/tree/selection

> The Tree component on the [docs](https://www.primefaces.org/primereact/showcase/#/tree) has the same issue
- This demo doesn't have selectable nodes. Therefore, the behavior in the demo is as expected.

Best Regards,
